tests/include/monolog.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
```

Comment:
2022?

Reply:
Addressed in [146d6ec](https://github.com/newrelic/newrelic-php-agent/pull/502/commits/146d6eccd9807e27350f6cb1bb1e53b2661f5054)